.circleci/config.yml

```diff
@@ -7,4 +7,16 @@ jobs:
     steps:
       - checkout
       - run: go get -v -t -d ./...
-      - run: go test -v ./...
\ No newline at end of file
+      - run: go test -v ./...
+
+      
+workflows:
+  version: 2
+  build:
+    jobs:
+      - build
```

Comment:
Not really a build job. Perhaps lets do test and lint?

Reply:
@ansermino we have to revisit this when we actually have some code and structure. There's definitely few more config to add.